box_it_script/box_it.js

```diff
@@ -0,0 +1,72 @@
+#!/home/nimbus-user/.nvm/versions/node/v17.6.0/bin/node
```

Comment:
JFYI. Defining path of node this way running `./box_it.js ...` only works off this specified computer.